launchlib/launcher_test.go

```diff
@@ -112,3 +112,22 @@ func TestKeysAreNotExpanded(t *testing.T) {
 		t.Errorf("Expected %%CWD%% to exist in map and not be expanded, but it didn't")
 	}
 }
+
+func TestMkdirChecksDirectorySyntax(t *testing.T) {
+	err := MkDirs([]string{"abc/def1"})
+	assert.NoError(t, err)
+
+	err = MkDirs([]string{"abc"})
+	assert.NoError(t, err)
+
+	require.NoError(t, os.RemoveAll("abc"))
+
+	err = MkDirs([]string{"^&*"})
+	assert.Error(t, err)
```

Comment:
If you're checking for a known error, typically better to assert something about the error message here (ensures that this test only passes if it fails for the reason you expect rather than for any reason)